### PR TITLE
Name of the class scheduler changed to Scheduler

### LINKED ISSUE #268: scheduler.py: Change classname to titlecase
Follow the [wiki](https://github.com/NIT-dgp/UIP/wiki) and make modifications accordingly.

- [ ] Rename the class `scheduler` to `Scheduler`
- [ ] Change the definitions in the class to match `snake_case`

### PR DESCRIPTION
**Pull request by vartika-arora**
Name of the class scheduler changed to Scheduler: scheduler.py,mainGui.py,UIP.py

class 'scheduler' changed to 'Scheduler' in all classes of UIP
Fixes #268

### CODE REVIEW

**Comment by abhsag24** [comment]
Hi thank you for your Patch make sure you have the right commit structure that is 
```
tag: Shortlog

body
Fixes <issue number/url>
```
Also please upload a screenshot if you have made changes to the gui.

**Comment by nkprince007** [comment]
@vartika-arora please follow the correct commit structure.

File: uiplib/UIP.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from daemoniker import Daemonizer
 
 from uiplib.settings import ParseSettings, HOME_DIR
-from uiplib.scheduler import scheduler
+from uiplib.scheduler import Scheduler
```
Comment (abhsag24) [inline uiplib/UIP.py]: yu also need to change the call to this class. in the file

File: uiplib/gui/mainGui.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 from threading import Thread, active_count
 
 from uiplib.gui import generalTab, settingsTab
-from uiplib.scheduler import scheduler
+from uiplib.scheduler import Scheduler
```
Comment (abhsag24) [inline uiplib/gui/mainGui.py]: u also need to change the call to this class. in the file

**Comment by abhsag24** [comment]
@vartika-arora that's a nice commit, u just need to mae one change and follow the proper commit message structure

**Comment by vartika-arora** [comment]
I have made changes to mainGui.py. Are there still errors in my commit message?

**Comment by abhsag24** [comment]
@vartika-arora yes you need two squash your two commits together. and just have 1 commit which says something like : `Rename scheduler to Scheduler` and also add a fixes tag with issue no. See `CONTRIBUTING.md` for more on what is a proper commit structure

**Comment by abhsag24** [comment]
Awesome work @vartika-arora :D

**Comment by abhsag24** [comment]
merging :)

**Comment by abhsag24** [comment]
@vartika-arora i just noticed, the fixes tag was stilll missing

**Comment by abhsag24** [comment]
it's alright for now. i'll close the issue manually, althugh do keep this in mind from next time :)

**Comment by vartika-arora** [comment]
Thank you for merging! And I'll keep that in mind from next time

**Pull request merged**